communication/mavlink_stream.c

```diff
@@ -52,21 +52,42 @@
 // PUBLIC FUNCTIONS IMPLEMENTATION
```

Comment:
Done in https://github.com/lis-epfl/MAVRIC_Library/commit/dc8733c205d9ee38d3dcbc955b44ee709e1db879